tests/unit/dataactcore/factories/staging.py

```diff
@@ -239,6 +239,7 @@ class Meta:
     business_funds_indicator = fuzzy.FuzzyText()
     business_types = fuzzy.FuzzyText()
     cfda_number = fuzzy.FuzzyText()
+    cfda_title = fuzzy.FuzzyText()
```

Comment:
It shouldn't be adding to the detached one, we aren't asking them to include the title in their file, we're deriving it on submit